src/sagemaker/serializers.py

```diff
@@ -241,3 +241,27 @@ def serialize(self, data):
         buffer = io.BytesIO()
         scipy.sparse.save_npz(buffer, data)
         return buffer.getvalue()
+
+
+class LibSVMSerializer(BaseSerializer):
+    """Searilize data of various formats to a LibSVM-formatted string."""
+
+    CONTENT_TYPE = "text/libsvm"
```

Comment:
Note: XGBoost uses the deprecated https://github.com/aws/sagemaker-containers for model serving. The newer SageMaker Inference Toolkit does not support the`"text/libsvm"` MIME type. When XGBoost gets migrated to the SageMaker Inference Toolkit, someone would need to add support for handling `"text/libsvm"`.

Reply:
👍